File: furniturewebdemo1/src/main/java/com/example/furniturewebdemo1/payload/AuthResponse.java
package com.example.furniturewebdemo1.payload;

import com.example.furniturewebdemo1.model.User;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

import java.util.Optional;

@Getter
@Setter
@AllArgsConstructor
public class AuthResponse {
    private String accessToken;
    private String tokenType = "Bearer";
    private User user;

    public AuthResponse(String accessToken) {
        this.accessToken = accessToken;
    }


    public AuthResponse( User user) {

        this.user = user;
    }


}

File: WebNoiThat-master/src/app/corecontrol/payloads/auth-response.ts
import { User } from '../models/user';

export class AuthResponse {
    accessToken: string;
    tokenType: string = "Bearer";
    user:User;

    // constructor(accessToken: string, tokenType: string, user: User) {
    //     this.accessToken = accessToken;
    //     this.tokenType = tokenType;
    //     this.user=user;
    // }
    //constructor(){}
    
}